src/components/screens/GuideScreen/Hero.stories.js

```diff
@@ -89,53 +94,95 @@ const storyLanguages = [
     tutorial: '/intro-to-storybook/react/ru/get-started/',
   },
 ];
-//
 // list of baseline language for the first 3 stories
-const defaultLanguages = storyLanguages.slice(0, 5);
-//
+export const defaultLanguagesData = storyLanguagesData.slice(0, 5);
 
-const props = {
+export const Basic = args => <Hero {...args} />;
+Basic.storyName = 'default';
+Basic.args = {
   ctaHref: '/get-started',
   description:
     "Visual testing is a pragmatic yet precise way to verify the look of UI components. It's practiced by companies like Slack, Lonely Planet and Walmart. This 31-page handbook gives you an overview of visual testing in React.",
   imagePath: '/guide-cover/intro.svg',
   themeColor: '#6F2CAC',
   title: 'Visual Testing Handbook',
+  languages: defaultLanguagesData,
 };
 
-storiesOf('Screens|GuideScreen/Hero', module)
-  .addParameters({ component: Hero })
-  .add('default', () => <Hero {...props} languages={defaultLanguages} />)
-  .add('with contributor count', () => (
-    <Hero {...props} contributorCount="34+" languages={defaultLanguages} />
-  ))
-  .add('with chapter count', () => (
-    <Hero {...props} contributorCount="34+" chapterCount={9} languages={defaultLanguages} />
-  ))
-  .add('with only one language', () => (
-    <Hero
-      {...props}
-      contributorCount="34+"
-      chapterCount={9}
-      languages={storyLanguages.slice(0, 1)}
-    />
-  ))
-  .add('with +5 languages', () => (
-    <Hero
-      {...props}
-      contributorCount="34+"
-      chapterCount={9}
-      languages={storyLanguages.slice(0, 7)}
-    />
-  ))
-  .add('with +10 languages', () => (
-    <Hero
-      {...props}
-      contributorCount="34+"
-      chapterCount={9}
-      languages={storyLanguages.slice(0, 12)}
-    />
-  ))
-  .add('with +20 languages', () => (
-    <Hero {...props} contributorCount="34+" chapterCount={9} languages={storyLanguages} />
-  ));
+export const WithContributorCount = Basic.bind();
+WithContributorCount.storyName = 'with contributor count';
+WithContributorCount.args = {
+  ctaHref: '/get-started',
+  description:
+    "Visual testing is a pragmatic yet precise way to verify the look of UI components. It's practiced by companies like Slack, Lonely Planet and Walmart. This 31-page handbook gives you an overview of visual testing in React.",
+  imagePath: '/guide-cover/intro.svg',
+  themeColor: '#6F2CAC',
+  title: 'Visual Testing Handbook',
+  contributorCount: '34+',
+  languages: defaultLanguagesData,
+};
+export const WithChapterCount = Basic.bind();
+WithChapterCount.storyName = 'with chapter count';
+WithChapterCount.args = {
+  ctaHref: '/get-started',
+  description:
+    "Visual testing is a pragmatic yet precise way to verify the look of UI components. It's practiced by companies like Slack, Lonely Planet and Walmart. This 31-page handbook gives you an overview of visual testing in React.",
```

Comment:
If you want to reuse an arg across stories the easiest thing to do it add it to the component's args:

```js
export default {
 ...
 args: {
   description: "...",
  }
}
```

It will apply to every story.

If you want to only use the same arg for a subset of stories, you can override it for the stories where you don't, OR you can simply re-use the JS object:

```js
Basic.args = {
  // ...
 description: "...",
};

// Just specify the new stuff. 
WithContributorCount.args = {
   ...Basic.args,
   contributorCount: '34+',
}
```